packages/sit-onyx/src/components/OnyxCheckboxGroup/OnyxCheckboxGroup.vue

```diff
@@ -83,6 +83,7 @@ const masterCheckboxState = computed<Partial<OnyxCheckboxProps>>(() => {
   padding: 0;
   border: none;
   max-width: max-content;
+  min-width: unset;
```

Comment:
may I ask why this is needed? :)

Reply:
Otherwise the `<figure>` element does not respect the width e.g. when setting it to 12rem or something